rollup.config.js

```diff
@@ -13,10 +12,8 @@ export default {
     preserveModules: true,
     preserveModulesRoot: "src",
   },
-  plugins: [nodeResolve(), typescript(), commonjs()],
-  onwarn: function (warning, rollupWarn) {
-    if (warning.code !== "THIS_IS_UNDEFINED") {
-      rollupWarn(warning);
-    }
-  },
+  plugins: [
+    nodeResolve(),
+    typescript()
```

Comment:
why did we remove the warning and commonjs?

Reply:
For the warning, I don't think it's valuable to suppress undefined warnings.

For commonJS, that plugin is converting commonJS to ES modules, but all of our modules are already ES so it is extra complexity and potentially a source for confusion and errors